3과 5의 공배수 구현

xkrwjdals opened the pull request:
# 구현 내용
- String three_and_five(int n) 라는 method를 통해 3과 5의 공배수라면 "fizzbuzz"라는 문자열을 반환합니다.
- 만약 아니라면, return integer.toString(n);을 수행합니다.

xkrwjdals committed:
feat : Create three_and_five method

Print fizzbuzz when integer n is common multiple of 3 and 5
Otherwise, print integer n

xkrwjdals committed:
refactor: change name of method

doyooon reviewed (changes requested):
메소드 만들지 말고 메인에다가 바로 작성해주세요!

xkrwjdals committed:
feat : Create three_and_five method

Print fizzbuzz when integer n is common multiple of 3 and 5
Otherwise, print integer n

doyooon reviewed (approved):
Nicely done!!!!

doyooon committed:
Merge branch 'main' into main
